src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -362,7 +362,7 @@ def get(self, request: HttpRequest, organization: Organization, trace_id: str) -
             sentry_sdk.set_tag("discover.trace-view.warning", "root.extra-found")
             logger.warning(
                 "discover.trace-view.root.extra-found",
-                {"extra_roots": len(roots), **warning_extra},
+                extra={"extra_roots": len(roots), **warning_extra},
```

Comment:
mypy doing its job :D (prior to this it was trying to format this dictionary into the message)